app/components/shared/Table/Table.tsx

```diff
@@ -127,22 +127,30 @@ export const Table = <T extends IdObj>({
               </tr>
             </thead>
             <tbody>
-              {paginatedData.map((item) => (
-                <tr key={item.id}>
-                  {Object.entries(removeIdFromObject(item)).map(([key, value]) => (
-                    <td key={key}>
-                      {typeof value === "object"
-                        ? (
-                            value as {
-                              value: string
-                              element: JSX.Element
-                            }
-                          ).element
-                        : value}
-                    </td>
-                  ))}
+              {paginatedData.length ? (
+                paginatedData.map((item) => (
+                  <tr key={item.id}>
+                    {Object.entries(removeIdFromObject(item)).map(([key, value]) => (
+                      <td key={key}>
+                        {typeof value === "object"
+                          ? (
+                              value as {
+                                value: string
+                                element: JSX.Element
+                              }
```

Comment:
This type of definition should be declared in the types definition of this file instead the render return.

Reply:
added to the last commit 🫡